cmd/minishift/cmd/oc_env.go

```diff
@@ -18,6 +18,10 @@ package cmd
 
 import (
 	"fmt"
+	"os"
```

Comment:
This just seems to re-arrange the import statements. I guess we can exclude this. Trying to keep this change set minimal

Reply:
+1